RawDataUnpacker/CMakeLists.txt

```diff
@@ -47,6 +47,21 @@ cet_build_plugin(RawDataUnpacker art::source
   art_root_io::TFileService_service
 )
 
+cet_build_plugin(SSDUnpacker art::source
+  LIBRARIES   PRIVATE
+  RawData
+  RawDataUnpacker
+  ChannelMap
+  RunHistory
+  ChannelMapService_service
+  ROOT::Core
+  ROOT::Tree
+  ROOT::Hist
+#  artdaq_core::artdaq-core_Data
```

Comment:
we should remove commented out code if not needed. not a show stopper for merging though